src/GafferScene/UDIMQuery.cpp

```diff
@@ -206,8 +207,32 @@ struct InfoDataAccumulator
 		const IECoreScene::MeshPrimitive *meshPrimitive = runTimeCast<const IECoreScene::MeshPrimitive>( object.get() );
 		if( !meshPrimitive ) return true;
 
-		auto uvs = meshPrimitive->variableIndexedView<IECore::V2fVectorData>( m_uvSet,  IECoreScene::PrimitiveVariable::FaceVarying );
-		if( !uvs ) return true;
+		// Kinda funny that with all this C++ gunk around, we still have to resort to using a pointers in the
+		// end because IndexedView doesn't support move semantics
```

Comment:
I don't understand this comment - why would IndexedView need to support move semantics? Move semantics are for objects that own resources, and IndexedView doesn't own anything. I have no idea why you need all this unique_ptr juggling down below, so let's figure out what we can do to make it unnecessary.

Reply:
Perhaps you just meant that IndexedViews couldn't be assigned? Here's a PR that should fix that :

https://github.com/ImageEngine/cortex/pull/893